packages/server-core/src/social/invite/invite.hooks.ts

```diff
@@ -45,14 +56,92 @@ import {
   inviteResolver
 } from './invite.resolvers'
 
+async function addSearch(context: HookContext<InviteService>) {
+  if (!context.params.query || !context.params.query.search) {
+    return
+  }
+
+  const search = context.params.query.search
+
+  context.params.query = {
+    ...context.params.query,
+    $or: [
+      {
+        inviteType: {
+          $like: '%' + search + '%'
+        }
+      },
+      {
+        passcode: {
+          $like: '%' + search + '%'
+        }
+      }
+    ]
+  }
+}
+
+function checkQueryType(type: InviteQuery['type']) {
+  return (context: HookContext<InviteService>) => context.params.query?.type === type
+}
+
+async function addInvitesReceived(context: HookContext<InviteService>) {
```

Comment:
This can be renamed to `handleInvitee`

This will make naming similar to how we have in `user.hook.ts` for `handleUserSearch`

Reply:
implemented!